pkg/controller/workload/workload_processor.go

```diff
@@ -466,9 +481,9 @@ func (p *Processor) handleWorkload(workload *workloadapi.Workload) error {
 		return err
 	}
 
-	// update frontend and backend bpf map
-	if err := p.updateWorkload(workload); err != nil {
-		log.Errorf("updateWorkload %s failed: %v", workload.Uid, err)
+	// 4. update workload in frontend map
+	if err := p.updateWorkloadInFrontendMap(workload); err != nil {
+		log.Errorf("updateWorkloadInFrontendMap %s failed: %v", workload.Uid, err)
```

Comment:
It is recommended that failed resource recycling be considered. Otherwise, residual BPF maps exist.

Reply:
what do you mean